org.hdfgroup.HDFView.yml

```
@@ -0,0 +1,121 @@
+app-id: org.hdfgroup.HDFView
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk/jvm/openjdk-17
+    ANT_HOME: /usr/lib/sdk/openjdk/ant
+finish-args:
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+  - name: hdf4
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DHDF4_BUILD_FORTRAN:BOOL=OFF
+      - -DHDF4_BUILD_JAVA:BOOL=ON
+      - -DHDF4_BUILD_EXAMPLES:BOOL=OFF
+      - -DHDF4_INSTALL_BIN_DIR:STRING=bin
+      - -DHDF4_INSTALL_LIB_DIR:STRING=lib
+      - -DHDF4_INSTALL_JAR_DIR:STRING=lib
+      - -DHDF4_INSTALL_INCLUDE_DIR:STRING=include
+      - -DHDF4_INSTALL_DATA_DIR:STRING=share
+      - -DHDF4_INSTALL_CMAKE_DIR:STRING=share/cmake
+      - -DBUILD_TESTING:BOOL=OFF
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+    sources:
+      - type: archive
+        url: https://support.hdfgroup.org/ftp/HDF/releases/HDF4.2.15/src/hdf-4.2.15.tar.bz2
+        sha256: bde035ef5a1cd5fdbd0a7f1fa5c17e98bbd599300189ac4d234f16e9bb7bcb12
+      - type: patch
+        path: hdf-ppc.patch
+      - type: patch
+        path: hdf-4.2.4-sparc.patch
+      - type: patch
+        path: hdf-s390.patch
+      - type: patch
+        path: hdf-arm.patch
+      - type: patch
+        path: hdf-aarch64.patch
```

```suggestion
      - type: patch
        paths
          - hdf-ppc.patch
          - hdf-4.2.4-sparc.patch
          - hdf-s390.patch
          - hdf-arm.patch
          - hdf-aarch64.patch
```

I'll assume you need all these unsupported achiecture patches to get the aarch64 patch.

(if only they just used stdint)

---

> I'll assume you need all these unsupported achiecture patches to get the aarch64 patch.

Indeed...